client/src/main/java/com/flipkart/flux/client/intercept/TaskInterceptor.java

```diff
@@ -162,7 +162,7 @@ private String generateStateIdentifier(Method method) {
         return method.getName();
     }
 
-    private String generateTaskIdentifier(Method method) {
+    public String generateTaskIdentifier(Method method) {
```

Comment:
biggest blunder :) my bad, thought we were calling some method which has the implementation in task interceptor itself, and added line note also saying we should move it to some util class. Ignore it.